frontend/benefit/applicant/src/components/errorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Container from 'shared/components/container/Container';
+
+import { $PrimaryButton } from '../applications/Applications.sc';
+import {
+  $ActionsContainer,
+  $ErrorPageContainer,
+  $ErrorPageMessage,
+  $ErrorPageTitle,
+  $IconAlertCircle,
+} from './ErrorPage.sc';
+import { useErrorPage } from './useErrorPage';
+
+export interface ErrorPageProps {
+  title: string;
+  message: string;
+  showActions?: boolean;
+}
+
+const ErrorPage: React.FC<ErrorPageProps> = ({
+  title,
+  message,
+  showActions = false,
+}) => {
+  const { t, handleBackClick } = useErrorPage();
+  return (
+    <Container>
+      <$ErrorPageContainer>
+        <$IconAlertCircle size="xl" />
+        <$ErrorPageTitle>{title}</$ErrorPageTitle>
+        <$ErrorPageMessage>{message}</$ErrorPageMessage>
+        {showActions && (
+          <$ActionsContainer>
+            <$PrimaryButton onClick={handleBackClick}>
+              {t('common:errorPage.back')}
+            </$PrimaryButton>
```

Comment:
Not sure if going back is the right action for the path that led to the error state, maybe refreshing or going home are safer options? 🤔

Reply:
back is just a name, the action is redirect to the root. I ll rename, so its more clear